### ISSUE TITLE
GAP 4.11.0 startup failure - dlopen issue on macOS 10.15.7 (in GAP package Browse)

### ISSUE THREAD

**Issue by dimpase** [conda-forge/gap-feedstock#41]
Issue: 
```
(base) dima@oucl13243 ~ % gap
 ┌───────┐   GAP 4.11.0 of 29-Feb-2020
 │  GAP  │   https://www.gap-system.org
 └───────┘   Architecture: 64
 Configuration:  gmp 6.2.0, GASMAN, readline
 Loading the library and packages ...
#W dlopen() error: dlopen(/Users/dima/opt/miniconda3/share/gap/pkg/Browse/bin/64/ncurses.so, 9): Library not loaded: @rpath/libpanel.6.dylib
  Referenced from: /Users/dima/opt/miniconda3/share/gap/pkg/Browse/bin/64/ncurses.so
  Reason: image not found
Error, module '/Users/dima/opt/miniconda3/share/gap/pkg/Browse/bin/64/ncurses.so' not found in
  LOAD_DYN( arg[1], false ) at /Users/dima/opt/miniconda3/share/gap/lib/files.gd:587 called from 
<function "LoadDynamicModule">( <arguments> )
 called from read-eval loop at /Users/dima/opt/miniconda3/share/gap/pkg/Browse/init.g:34
Error, was not in any namespace at /Users/dima/opt/miniconda3/share/gap/lib/variable.g:292 called from
LEAVE_NAMESPACE(  ); at /Users/dima/opt/miniconda3/share/gap/lib/package.gi:1303 called from
ReadPackage( pkgname, "init.g" ); at /Users/dima/opt/miniconda3/share/gap/lib/package.gi:1670 called from
LoadPackage( name, false ) at /Users/dima/opt/miniconda3/share/gap/lib/package.gi:1980 called from
func(  ); at /Users/dima/opt/miniconda3/share/gap/lib/system.g:220 called from
<function "CallAndInstallPostRestore">( <arguments> )
 called from read-eval loop at /Users/dima/opt/miniconda3/share/gap/lib/init.g:733
you can 'quit;' to quit to outer loop, or
you can 'return;' to continue
brk> 
```

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# packages in environment at /Users/dima/opt/miniconda3:
#
# Name                    Version                   Build  Channel
appnope                   0.1.0           py38h50d1736_1002    conda-forge
arb                       2.18.0               h375d61f_0    conda-forge
argon2-cffi               20.1.0           py38h4d0b108_2    conda-forge
async_generator           1.10                       py_0    conda-forge
attrs                     20.3.0             pyhd3deb0d_0    conda-forge
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                        py_2    conda-forge
backports.functools_lru_cache 1.6.1                      py_0    conda-forge
bleach                    3.2.1              pyh9f0ad1d_0    conda-forge
bliss                     0.73.0.debian_1.sage_2016_08_02      h879752b_0    conda-forge
bzip2                     1.0.8                haf1e3a3_3    conda-forge
c-ares                    1.16.1               haf1e3a3_3    conda-forge
ca-certificates           2020.11.8            h033912b_0    conda-forge
certifi                   2020.11.8        py38h50d1736_0    conda-forge
cffi                      1.14.0           py38hc512035_1  
chardet                   3.0.4                 py38_1003  
conda                     4.9.0            py38h5347e94_1    conda-forge
conda-package-handling    1.6.1            py38h1de35cc_0  
cryptography              2.9.2            py38ha12b0ac_0  
decorator                 4.4.2                      py_0    conda-forge
defusedxml                0.6.0                      py_0    conda-forge
e-antic                   0.1.7                h49de2a9_1    conda-forge
entrypoints               0.3             py38h32f6830_1002    conda-forge
fmt                       6.2.1                ha1b3eb9_0    conda-forge
gap                       4.11.0               h61dea56_2    conda-forge
gap-core                  4.11.0               h74c63e5_2    conda-forge
gap-defaults              4.11.0                        2    conda-forge
gf2x                      1.3.0                h7e0514a_1    conda-forge
gmp                       6.2.0                h2e338ed_4    conda-forge
icu                       67.1                 hb1e8313_0    conda-forge
idna                      2.9                        py_1  
importlib-metadata        2.0.0            py38h32f6830_0    conda-forge
importlib_metadata        2.0.0                         1    conda-forge
ipykernel                 5.3.4            py38h1cdfbd6_1    conda-forge
ipython                   7.19.0           py38h9bb44b7_0    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
jedi                      0.17.2           py38h50d1736_1    conda-forge
jinja2                    2.11.2             pyh9f0ad1d_0    conda-forge
json5                     0.9.5              pyh9f0ad1d_0    conda-forge
jsonschema                3.2.0            py38h32f6830_1    conda-forge
jupyter_client            6.1.7                      py_0    conda-forge
jupyter_core              4.6.3            py38h50d1736_2    conda-forge
jupyterlab                2.2.9                      py_0    conda-forge
jupyterlab_pygments       0.1.2              pyh9f0ad1d_0    conda-forge
jupyterlab_server         1.2.0                      py_0    conda-forge
krb5                      1.17.1               hddcf347_0  
libarchive                3.3.3             hf12134e_1008    conda-forge
libcurl                   7.71.1               h9bf37e3_8    conda-forge
libcxx                    11.0.0               h439d374_0    conda-forge
libedit                   3.1.20181209         hb402a30_0  
libev                     4.33                 haf1e3a3_1    conda-forge
libffi                    3.3                  h0a44026_1  
libflint                  2.6.3                haaaf077_0    conda-forge
libiconv                  1.16                 haf1e3a3_0    conda-forge
libnghttp2                1.41.0               h7580e61_2    conda-forge
libsemigroups             1.0.9                h4a8c4bd_1    conda-forge
libsodium                 1.0.18               haf1e3a3_1    conda-forge
libsolv                   0.7.16               hab81aa3_0    conda-forge
libssh2                   1.9.0                h8a08a2b_5    conda-forge
libxml2                   2.9.10               h2c6e4a5_2    conda-forge
llvm-openmp               11.0.0               h73239a0_1    conda-forge
lz4-c                     1.9.2                hb1e8313_3    conda-forge
lzo                       2.10              haf1e3a3_1000    conda-forge
mamba                     0.6.4            py38hcc219c9_0    conda-forge
markupsafe                1.1.1            py38h94c058a_2    conda-forge
mistune                   0.8.4           py38h7e3306e_1002    conda-forge
mpc                       1.1.0             ha57cd0f_1009    conda-forge
mpfi                      1.5.4             h4160ff4_1001    conda-forge
mpfr                      4.0.2                h72d8aaf_1    conda-forge
nbclient                  0.5.1                      py_0    conda-forge
nbconvert                 6.0.7            py38h32f6830_2    conda-forge
nbformat                  5.0.8                      py_0    conda-forge
ncurses                   6.2                  h0a44026_1  
nest-asyncio              1.4.2              pyhd8ed1ab_0    conda-forge
normaliz                  3.8.5                h88f7564_0    conda-forge
notebook                  6.1.5            py38h50d1736_0    conda-forge
ntl                       11.4.3               h2295e1f_1    conda-forge
openssl                   1.1.1h               haf1e3a3_0    conda-forge
packaging                 20.4               pyh9f0ad1d_0    conda-forge
pandoc                    2.11.1.1             hbcb3906_0    conda-forge
pandocfilters             1.4.3            py38hecd8cb5_1  
pari                      2.11.4          pl5262h250645c_0    conda-forge
pari-galdata              0.0.20180411                  0    conda-forge
pari-seadata-small        0.0.20090618                  0    conda-forge
parso                     0.7.1              pyh9f0ad1d_0    conda-forge
perl                      5.26.2            hbcb3906_1008    conda-forge
pexpect                   4.8.0            py38h32f6830_1    conda-forge
pickleshare               0.7.5           py38h32f6830_1002    conda-forge
pip                       20.0.2                   py38_3  
planarity                 3.0.0.5           h1de35cc_1002    conda-forge
prometheus_client         0.8.0              pyh9f0ad1d_0    conda-forge
prompt-toolkit            3.0.8              pyha770c72_0    conda-forge
ptyprocess                0.6.0                   py_1001    conda-forge
pycosat                   0.6.3            py38h1de35cc_1  
pycparser                 2.20                       py_0  
pygments                  2.7.2                      py_0    conda-forge
pyopenssl                 19.1.0                   py38_0  
pyparsing                 2.4.7              pyh9f0ad1d_0    conda-forge
pyrsistent                0.17.3           py38h7e3306e_1    conda-forge
pysocks                   1.7.1                    py38_0  
python                    3.8.3                h26836e1_1  
python-dateutil           2.8.1                      py_0    conda-forge
python.app                2                       py38_10  
python_abi                3.8                      1_cp38    conda-forge
pyzmq                     19.0.2           py38h2c785a9_2    conda-forge
readline                  8.0                  h1de35cc_0  
requests                  2.23.0                   py38_0  
ruamel_yaml               0.15.87          py38h1de35cc_0  
send2trash                1.5.0                      py_0    conda-forge
setuptools                47.1.1                   py38_0  
six                       1.15.0                     py_0  
sqlite                    3.31.1               h5c1f38d_1  
terminado                 0.9.1            py38h50d1736_1    conda-forge
testpath                  0.4.4                      py_0    conda-forge
tk                        8.6.8                ha441bb4_0  
tornado                   6.1              py38h7e3306e_0    conda-forge
tqdm                      4.46.0                     py_0  
traitlets                 5.0.5                      py_0    conda-forge
urllib3                   1.25.8                   py38_0  
wcwidth                   0.2.5              pyh9f0ad1d_2    conda-forge
webencodings              0.5.1                      py_1    conda-forge
wheel                     0.34.2                   py38_0  
xz                        5.2.5                h1de35cc_0  
yaml                      0.1.7                hc338f04_2  
zeromq                    4.3.3                hb1e8313_2    conda-forge
zipp                      3.4.0                      py_0    conda-forge
zlib                      1.2.11               h1de35cc_3  
zstd                      1.4.5                h289c70a_2    conda-forge
(base) dima@oucl13243 ~ % 

```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info
     active environment : base
    active env location : /Users/dima/opt/miniconda3
            shell level : 1
       user config file : /Users/dima/.condarc
 populated config files : /Users/dima/.condarc
          conda version : 4.9.0
    conda-build version : not installed
         python version : 3.8.3.final.0
       virtual packages : __osx=10.15.7=0
                          __unix=0=0
                          __archspec=1=x86_64
       base environment : /Users/dima/opt/miniconda3  (writable)
           channel URLs : https://conda.anaconda.org/conda-forge/osx-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /Users/dima/opt/miniconda3/pkgs
                          /Users/dima/.conda/pkgs
       envs directories : /Users/dima/opt/miniconda3/envs
                          /Users/dima/.conda/envs
               platform : osx-64
             user-agent : conda/4.9.0 requests/2.23.0 CPython/3.8.3 Darwin/19.6.0 OSX/10.15.7
                UID:GID : 505:20
             netrc file : None
           offline mode : False
```
</details>

**Comment by isuruf** [comment]
`ncurses` is coming from `defaults` instead of `conda-forge`.

Can you run a `conda update --all`?

**Comment by dimpase** [comment]
after `conda update --all` GAP starts, but breaks upon loading `semigroups` package (the breakage is even earlier, in `digraphs`):
```
% gap
 ┌───────┐   GAP 4.11.0 of 29-Feb-2020
 │  GAP  │   https://www.gap-system.org
 └───────┘   Architecture: 64
 Configuration:  gmp 6.2.0, GASMAN, readline
 Loading the library and packages ...
 Packages:   AClib 1.3.2, Alnuth 3.1.2, AtlasRep 2.1.0, AutoDoc 2019.09.04, AutPGrp 1.10.2, Browse 1.8.8, CaratInterface 2.3.3, CRISP 1.4.5, Cryst 4.1.23, CrystCat 1.1.9, CTblLib 1.2.2, FactInt 1.6.3, 
             FGA 1.4.0, Forms 1.2.5, GAPDoc 1.6.3, genss 1.6.6, IO 4.7.0, IRREDSOL 1.4, LAGUNA 3.9.3, orb 4.8.3, Polenta 1.3.9, Polycyclic 2.15.1, PrimGrp 3.4.0, RadiRoot 2.8, recog 1.3.2, 
             ResClasses 4.7.2, SmallGrp 1.4.1, Sophus 1.24, SpinSym 1.5.2, TomLib 1.2.9, TransGrp 2.0.5, utils 0.69
 Try '??help' for help. See also '?copyright', '?cite' and '?authors'
gap> LoadPackage("semigroups");
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
Loading  GRAPE 4.8.3 (GRaph Algorithms using PErmutation groups)
by Leonard H. Soicher (http://www.maths.qmul.ac.uk/~lsoicher/).
Homepage: https://gap-packages.github.io/grape
Report issues at https://github.com/gap-packages/grape/issues
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
#W dlopen() error: dlopen(/Users/dima/opt/miniconda3/share/gap/pkg/digraphs-1.1.1/bin/64/digraphs.so, 9): Library not loaded: /Users/runner/miniforge3/conda-bld/gap-split_1603138614927/work/pkg/digraphs\
-1.1.1/extern/edge-addition-planarity-suite-Version_3.0.0.5/../../bin/lib/libplanarity.0.dylib
  Referenced from: /Users/dima/opt/miniconda3/share/gap/pkg/digraphs-1.1.1/bin/64/digraphs.so
  Reason: image not found
Error, module '/Users/dima/opt/miniconda3/share/gap/pkg/digraphs-1.1.1/bin/64/digraphs.so' not found in
  LOAD_DYN( arg[1], false ) at /Users/dima/opt/miniconda3/share/gap/lib/files.gd:587 called from 
<function "LoadDynamicModule">( <arguments> )
 called from read-eval loop at /Users/dima/opt/miniconda3/share/gap/pkg/digraphs-1.1.1/init.g:24
Error, was not in any namespace at /Users/dima/opt/miniconda3/share/gap/lib/variable.g:292 called from
LEAVE_NAMESPACE(  ); at /Users/dima/opt/miniconda3/share/gap/lib/package.gi:1303 called from
ReadPackage( pkgname, "init.g" ); at /Users/dima/opt/miniconda3/share/gap/lib/package.gi:1670 called from
<function "LoadPackage">( <arguments> )
 called from read-eval loop at *stdin*:1
you can 'quit;' to quit to outer loop, or
you can 'return;' to continue
brk> 
```

**Comment by dimpase** [comment]
here is a minimal example:
```
% gap
 ┌───────┐   GAP 4.11.0 of 29-Feb-2020
 │  GAP  │   https://www.gap-system.org
 └───────┘   Architecture: 64
 Configuration:  gmp 6.2.0, GASMAN, readline
 Loading the library and packages ...
 Packages:   AClib 1.3.2, Alnuth 3.1.2, AtlasRep 2.1.0, AutoDoc 2019.09.04, AutPGrp 1.10.2, Browse 1.8.8, CaratInterface 2.3.3, CRISP 1.4.5, Cryst 4.1.23, CrystCat 1.1.9, CTblLib 1.2.2, FactInt 1.6.3, 
             FGA 1.4.0, Forms 1.2.5, GAPDoc 1.6.3, genss 1.6.6, IO 4.7.0, IRREDSOL 1.4, LAGUNA 3.9.3, orb 4.8.3, Polenta 1.3.9, Polycyclic 2.15.1, PrimGrp 3.4.0, RadiRoot 2.8, recog 1.3.2, 
             ResClasses 4.7.2, SmallGrp 1.4.1, Sophus 1.24, SpinSym 1.5.2, TomLib 1.2.9, TransGrp 2.0.5, utils 0.69
 Try '??help' for help. See also '?copyright', '?cite' and '?authors'
gap> LoadPackage("digraphs");
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
Loading  GRAPE 4.8.3 (GRaph Algorithms using PErmutation groups)
by Leonard H. Soicher (http://www.maths.qmul.ac.uk/~lsoicher/).
Homepage: https://gap-packages.github.io/grape
Report issues at https://github.com/gap-packages/grape/issues
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
#W dlopen() error: dlopen(/Users/dima/opt/miniconda3/share/gap/pkg/digraphs-1.1.1/bin/64/digraphs.so, 9): Library not loaded: /Users/runner/miniforge3/conda-bld/gap-split_1603138614927/work/pkg/digraphs\
-1.1.1/extern/edge-addition-planarity-suite-Version_3.0.0.5/../../bin/lib/libplanarity.0.dylib
  Referenced from: /Users/dima/opt/miniconda3/share/gap/pkg/digraphs-1.1.1/bin/64/digraphs.so
  Reason: image not found
Error, module '/Users/dima/opt/miniconda3/share/gap/pkg/digraphs-1.1.1/bin/64/digraphs.so' not found in
  LOAD_DYN( arg[1], false ) at /Users/dima/opt/miniconda3/share/gap/lib/files.gd:587 called from 
<function "LoadDynamicModule">( <arguments> )
 called from read-eval loop at /Users/dima/opt/miniconda3/share/gap/pkg/digraphs-1.1.1/init.g:24
Error, was not in any namespace at /Users/dima/opt/miniconda3/share/gap/lib/variable.g:292 called from
LEAVE_NAMESPACE(  ); at /Users/dima/opt/miniconda3/share/gap/lib/package.gi:1303 called from
ReadPackage( pkgname, "init.g" ); at /Users/dima/opt/miniconda3/share/gap/lib/package.gi:1670 called from
<function "LoadPackage">( <arguments> )
 called from read-eval loop at *stdin*:1
you can 'quit;' to quit to outer loop, or
you can 'return;' to continue
brk> 
```

**Comment by isuruf** [comment]
Check in 1.5 hours. Fixed both gap 4.11 and 4.10
